welcome_to_jsk_fetch/welcome-to-jsk.l

```diff
@@ -122,17 +122,15 @@
 )
 
 
-(defun main ()
-  (load "package://fetcheus/fetch-interface.l")
-
-  (fetch-init)
-
+(defun nav ()
   (ros::advertise "go_to_point_arrived" std_msgs::string 1)
   (ros::ros-info "[Get ready to subscribe: /go_to_point]")
 
   (ros::subscribe "go_to_point" std_msgs::string #'go-to-point)
   (ros::spin)
   )
 
-
-(main)
+(defun init()
+  (load "package://fetcheus/fetch-interface.l")
```

Comment:
why you load fetch-interface here. you already load at  https://github.com/708yamaguchi/jsk_demos/blob/1293265accd0d834f361b9901728d22211dea9ff/welcome_to_jsk_fetch/welcome-to-jsk.l#L3

Reply:
did you add undock action?

Reply:
At my latest commit, I remove unnecessary `(load "package://fetcheus/fetch-interface.l")`.
Also, undock fetch at the start of demo, if fetch is docking.